nncf/compression_method_api.py

```diff
@@ -202,7 +203,9 @@ def run_batchnorm_adaptation(self, config):
                     'as the adaptation data loader is not provided as an extra struct. '
                     'Refer to `NNCFConfig.register_extra_structs` and the `BNAdaptationInitArgs` class')
                 return
-
+            batch_size = bn_adaptation_args.data_loader.batch_size
+            num_bn_forget_steps = numpy.ceil(num_bn_forget_samples / batch_size)
+            num_bn_adaptation_steps = numpy.ceil(num_bn_adaptation_samples / batch_size)
             bn_adaptation_runner = DataLoaderBNAdaptationRunner(self._model, bn_adaptation_args.device,
```

Comment:
Since you decided to take this implicit approach, you should document EVERYWHERE that the actual number of samples will be a closest multiple of the `InitializingDataLoader`'s batch size, and not exactly the number that the user specified in the config file.